mu/interface.py

```diff
@@ -809,16 +813,55 @@ def process_bytes(self, bs):
         # then move it there.
         while tc.movePosition(QTextCursor.Down):
             pass
-        for b in bs:
-            if b == 8:  # \b
+
+        i = 0
+        while i < len(data):
+            if data[i] == 8:  # \b
                 tc.movePosition(QTextCursor.Left)
                 self.setTextCursor(tc)
-            elif b == 13:  # \r
+            elif data[i] == 13:  # \r
                 pass
+            elif data[i] == 27 and data[i+1] == 91:  # VT100 cursor: <Esc>[
```

Comment:
From the great Travis: `E226 missing whitespace around arithmetic operator`

I'm assuming it wants you to use `data[i + 1]`  in place of `data[i+1]`